app/src/main/java/org/p2p/wallet/history/ui/history/HistoryFragment.kt

```diff
@@ -38,7 +39,7 @@ class HistoryFragment :
     }
 
     override fun onUserSendLinksClicked() {
-        replaceFragment(HistorySendLinksFragment.create())
+        addFragment(HistorySendLinksFragment.create())
```

Comment:
why we need to use add here?

Reply:
to prevent destroying the HistoryFragment, we don't need to reload the transaction history from scratch as a result

Reply:
I propose to add check in `presenter` if history is not null or empty, then we don't load the history. 
And we will still be able to leave this as `replace`

Reply:
@1sakov The HistoryListPresenter is destroyed when we replace the HistoryFragment with the links fragment because it attaches directly to the HistoryListView. As the HistoryListView is also being destroyed, we can't simply check whether the data has already been loaded or not within the presenter. Or am I missing something?

Reply:
Decided to move the HistoryListPresenter back to the HistoryFragment as it was before, cause the presenter should survive destroying HistoryListView